orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RollingRedBlackStrategy.groovy

```diff
@@ -147,6 +137,7 @@ class RollingRedBlackStrategy implements Strategy, ApplicationContextAware {
         )
       }
 
+      // TODO-AJ this has questionable behavior w/ > 1 old server group
```

Comment:
This is a cluster-wide operation whereas you're resizing based on a single server group. 

I would think that we would be disabling percentages of a specific server group and not the overall cluster. Specifically noticed this getting weird when I had more then one active server group. 

Am I missing something?

Reply:
I figured we'd want this to disable everything but what we want to deploy. Similar to how red/black uses a cluster wide disable. Does that make sense?

Reply:
I think the bit that confused me is that I saw (or at least I thought I saw) it happened after the first % rollout completed. That struck me as a little weird is all.

Reply:
Huh do you mean it disabled something incorrectly after the new SG grew for the first time?

Reply:
Hrmph. Just ignore me ... at this point not seeing any premature disables, so that's good.

Every pre-existing server group gets the disable percentage treatment , that seems alright as well.